scripts/travis_install.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+# Copyright 2017 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# A script to perform the download and installation of protobuf.
+
+set -e
+
+echo "Downloading protoc."
+mkdir -p $PROTOC_DOWNLOAD_PATH
+cd $PROTOC_DOWNLOAD_PATH
+wget https://github.com/google/protobuf/releases/download/v3.2.0/protoc-3.2.0-linux-x86_64.zip
```

Comment:
nit: Please feel free to pass on this, or do it later.  My nit is that we have 2 places where we are specifying which protoc to download.  So, we might update one, but forget to update here.  So, is there anyway that we can specify the protoc once, and both this and forseti-instance.py would be updated?  Again, not critical or blocking.

https://github.com/GoogleCloudPlatform/forseti-security/blob/91621f3109d3339b1da9a1eb08e7fdbe36ecc692/deployment-templates/py/forseti-instance.py#L121